nbodykit/measurestats.py

```diff
@@ -481,7 +481,7 @@ def project_to_basis(comm, x3d, y3d, edges, los='z', poles=[], symmetric=True):
                     weighted_y3d.real[:, nonsingular] *= 2.
                     weighted_y3d.imag[:, nonsingular] = 0.
                     
-            weighted_y3d *= 0.5*(2*ell + 1.)
+            weighted_y3d *= (2.*ell + 1.)
```

Comment:
I think the error is still sqrt(Nmodes) . We are counting all of the modes in the box, which I think is the number you want to use, since the power at mu and -mu doesn't have to be the same in the most general case (I think). 

I have verified that this change reproduces the same power as before for the test cases, although the mode counting produces a different result. I think the version now is correct: the number of modes in a k-bin of width dk agrees with the analytic estimate N = 4π k^2 ∆k V / (2π)3.